webcodecs/audioDecoder-codec-specific.https.any.js

```diff
@@ -257,7 +298,7 @@ promise_test(async t => {
   });
 
   await decoder.flush();
-  assert_equals(outputs, CHUNKS.length, 'outputs');
+  assert_equals(outputs, CONFIG.codec === 'vorbis' ? CHUNKS.length - 1 : CHUNKS.length, 'outputs');
```

Comment:
Remove those special cases throughout (and maybe file a bug on Chromium's tracker?).

Reply:
I think emitting one packet less when decoding vorbis is actually the corrrect behavior since the first vorbis packet produces no samples.